### PR TITLE
Define electra types and apis 

### PR DESCRIPTION
**Pull request by ensi321**
Add electra type definition and update apis to include new types.

Also update beacon apis submodule commit to `02d595daf39943430f54aa978f88a5c50d957e64`

Follow up of #101

### CODE REVIEW

File: apis/builder/blinded_blocks.yaml [hunk 1]
```diff
@@ -50,7 +53,7 @@ post:
             properties:
               version:
                 type: string
-                enum: [ bellatrix, capella, deneb ]
+                enum: [ bellatrix, capella, deneb, electra ]
                 example: "bellatrix"
               data:
                 oneOf:
```
Comment (ralexstokes) [inline apis/builder/blinded_blocks.yaml:59]: do we want to extend the `examples` below here w/ the electra one?

**Review by ralexstokes — COMMENTED** [review]
had one question but otherwise this looks great, thank you!!

if we update the `beacon-apis` submodule to at least `492bbbdbcad45249d9a8cd8c23e902c5dfe4ad28` then I think this will be final for pectra broadly

**Comment by ralexstokes** [comment]
also looks like the submodule update would help the linter errors

File: apis/builder/header.yaml [hunk 2]
```diff
@@ -44,20 +44,21 @@ get:
...
                  - $ref: "../../builder-oapi.yaml#/components/schemas/Capella.SignedBuilderBid"
                  - $ref: "../../builder-oapi.yaml#/components/schemas/Deneb.SignedBuilderBid"
+                 - $ref: "../../builder-oapi.yaml#/components/schemas/Electra.SignedBuilderBid"
           examples:
             bellatrix:
               $ref: "../../builder-oapi.yaml#/components/examples/Bellatrix.SignedBuilderBid"
             capella:
               $ref: "../../builder-oapi.yaml#/components/examples/Capella.SignedBuilderBid"
             deneb:
-              $ref: "../../builder-oapi.yaml#/components/examples/Deneb.SignedBuilderBid"
```
Comment (nflaig) [inline apis/builder/header.yaml:60]: this looks wrong, shouldn't there be a new example for `electra`

**Review by ralexstokes — APPROVED** [review]
lgtm, thanks!

**Pull request merged**